docs/rules/no-large-snapshots.md

```diff
@@ -0,0 +1,96 @@
+# disallow large snapshots (no-large-snapshots)
+
+When using Jest's snapshot capability one should be mindful of the size of
+created snapshots. As a best practice snapshots should be limited in size in
+order to be more manageable and reviewable. A stored snapshot is only as good as
+its review and as such keeping it short, sweet, and readable is important to
+allow for thorough reviews.
+
+## Rule Details
+
+This rule looks at all Jest snapshot files (files with `.snap` extension) and
```

Comment:
is it possible to ignore any of them? without using `overrides` in the config

Reply:
Yes it does work. But on snapshot updates it would be wiped.

Reply:
Should we include some sort of config then, to be able to ignore specific snapshots?

Reply:
Specific snapshots or specific snapshot files? If it is the latter then I am not sure how much value that would add, we would just be duplicating eslint functionality. If you mean ignoring specific snapshots then yes I believe that can be done pretty easily, I can work on that today.

Reply:
I meant snapshot files, not specific snapshots within them 🙂

Reply:
Thinking about it more I am not sure that being able to ignore a specific snapshot through a config would be a good user experience. They would have to provide the snapshot names in an array which could get unwieldy. It may be better to ignore whole files using built in `eslint` functionality.

How are you envisioning it?

Reply:
Yeah, `.eslintignore` is probably enough :)